deps/libmediasoupclient/include/Logger.hpp

```diff
@@ -52,7 +52,11 @@ namespace mediasoupclient
 	class Logger
 	{
 	public:
-		enum class LogLevel : uint8_t
+#ifdef __linux__
+		enum class LogLevel : u_int8_t
+#else
+        enum class LogLevel : uint8_t
+#endif
```

Comment:
We use uint8_t extensively in mediasoup server and of course it compiles in Linux, I just don't get this.

Reply:
I will check why in the broadcaster does not work.

Reply:
This was fixed long time ago in libmediasoupclient and recently added to this repo. Now we get libmediasoupclient from github instead of having it embedded in this repository.